onedal/spmd/_base.py

```diff
@@ -23,7 +23,7 @@
 
 
 class BaseEstimatorSPMD(ABC):
-    def _get_backend(self, module, submodule, method, *args, **kwargs):
+    def _get_backend(self, module, submodule=None, method=None, *args, **kwargs):
```

Comment:
No need adding this fix into current PR.
It is better to create separete PR with changes as well for https://github.com/intel/scikit-learn-intelex/blob/5a8f070abc98e4e33bf66fa24e058faeeb4981df/onedal/common/_base.py#L24
https://github.com/intel/scikit-learn-intelex/blob/5a8f070abc98e4e33bf66fa24e058faeeb4981df/onedal/common/_base.py#L33

and https://github.com/intel/scikit-learn-intelex/blob/5a8f070abc98e4e33bf66fa24e058faeeb4981df/onedal/spmd/_base.py#L26

Reply:
https://github.com/intel/scikit-learn-intelex/pull/1754